e2e/no-parallel-reflection-resolver/rector.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
+use Rector\Core\Configuration\Option;
+use Rector\DeadCode\Rector\Property\RemoveUnusedPrivatePropertyRector;
+
+return static function (ContainerConfigurator $containerConfigurator): void {
+    $parameters = $containerConfigurator->parameters();
+
+    $parameters->set(Option::PARALLEL, false);
+    $parameters->set(Option::PATHS, [
+        __DIR__ . '/src/',
+    ]);
+
+    $services = $containerConfigurator->services();
+    $services->set(RemoveUnusedPrivatePropertyRector::class);
+};
+
```

Comment:
I added one more test case, with parallel = false, so we know the results are for both identical.